addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -97,6 +97,11 @@ def _prepare_mail_values(self, slide_channel_partner):
             'recipient_ids': [(4, slide_channel_partner.partner_id.id)]
         }
 
+        # if user wants to override FROM by configuring the email template
```

Comment:
Hey @pyduf 

Looking over your feedback I have to admit that that your solution seems cleaner and more in sync with the already established approach in the function. Heck, I am facepalming for not thinking about it ;)

* For the `email_from` rendering, I agree that the ` _render_field` seems the best approach. I completely missed the mail.mail error because I was mostly testing on 15.0 where the default for the template was not set to `{{ user.email_formatted }}` but empty (thus returning a False by default).
But after testing it locally, your approach covers 15.0 and 16+
* For the the `reply_to`: I agree that the help string does not officially mention placeholders. But my guess is that curious users might still use variations of `{{ user.email_formatted }}` to define the field dynamically. In that case, if we use `'reply_to': self.template_id.reply_to or self.env.user.email_formatted`, the send email will  set `reply-to: user.email_formatted` in the email header. **BUT** this time it will not generate an error on Odoo's side, but might cause issues on the recipients side.
Thus, should we also apply the same logic and implicitly support dynamic placeholders for the reply_to field by using the `_render_field` method?